_includes/live-demos/mentions/index.html

```diff
@@ -1,6 +1,4 @@
 <textarea id="mentions">
-  <p>Type @ followed by a name</p>
-  <p>Aliquam purus massa, consequat ut massa euismod, vulputate vestibulum lorem. Curabitur ante ipsum, consectetur ut turpis ut, facilisis volutpat nulla. Etiam rhoncus sagittis urna, vitae interdum enim facilisis eget. Ut non tellus sit amet massa cursus tempus. Suspendisse mattis dui purus. Integer porta mi nec faucibus condimentum. Aliquam feugiat lacinia ante sagittis efficitur. Suspendisse ac porta tellus, nec feugiat libero.</p>
-  <p>Morbi pellentesque tortor sed cursus tempus. Nam commodo varius dapibus. Sed nec justo vel orci ornare porta. Nullam bibendum malesuada elit, eget bibendum arcu dignissim ac. Aliquam ac accumsan lectus. Aliquam odio sem, tincidunt sed aliquet quis, laoreet at diam. Donec mattis sapien at magna venenatis, id condimentum velit porta. Proin a augue eget urna volutpat bibendum. Praesent finibus urna leo, a euismod neque vestibulum eget. Aliquam iaculis turpis non libero suscipit, vitae cursus lectus rutrum. Quisque faucibus dignissim enim vitae blandit. Morbi ullamcorper et nulla quis varius. Vestibulum eu velit nec mauris ullamcorper venenatis. Suspendisse dictum vitae lectus at laoreet. Nullam ac volutpat arcu. Nulla vitae aliquam nibh.</p>
-  <p>Donec in tellus venenatis, aliquam felis eget, condimentum tellus. Praesent a scelerisque nibh. Sed efficitur lectus ut nisl mollis elementum. Aliquam bibendum, nisl a euismod dignissim, libero mi posuere elit, in fermentum mauris mi in nulla. Aliquam mollis rhoncus vestibulum. Fusce euismod ex ex, et ultrices nibh dignissim et. In hac habitasse platea dictumst. Donec in diam sit amet lorem condimentum consequat sit amet at lorem. Sed tempor sagittis mauris, ut placerat mi convallis in.</p>   
+  <p>Type "<kbd>@</kbd>" followed immediately by one or more characters.</p>
+  <p>For example: @a</p>
```

Comment:
We don't need this any more?

Reply:
Lorem Ipsum in demos === bad

We replaced it with end-user instructions, 'cos we can.